tests/e2e/Adapter/Scopes/SpatialTests.php

```
@@ -0,0 +1,1417 @@
+
+            // Test failure case: square should NOT contain distant point
+            if ($database->getAdapter()->getSupportForBoundaryInclusiveContains()) {
+                $distantPointSquare = $database->find($collectionName, [
+                    Query::contains('square', [[100, 100]]) // Point far outside square
+                ], Database::PERMISSION_READ);
+                $this->assertEmpty($distantPointSquare);
+            }
+
+            // Test failure case: square should NOT contain point on boundary
+            if ($database->getAdapter()->getSupportForBoundaryInclusiveContains()) {
+                $boundaryPointSquare = $database->find($collectionName, [
```

PHPMD (2.15.0)

Avoid unused local variables such as '$boundaryPointSquare'. (Unused Code Rules)

(UnusedLocalVariable)